tests/test_bot.py

```diff
@@ -343,7 +343,8 @@ def test_send_message(self):
         self.bot.slack_client.api_call.assert_called_with('chat.postMessage',
                                                           channel='channel_id',
                                                           text='Hi test',
-                                                          as_user=True)
+                                                          as_user=True,
+                                                          attachments='null')
```

Comment:
Does this have to be set to 'null' for the set to pass? If so something seems a little off (we shouldn't expect end users to pass attachments='null' everywhere). The tests are the messiest part of this project (I'm hoping to rework them, but just need to find the time), so if it is required it's likely the test is wrong and needs tweaking. I'm more than happy to help with this if needed.

Reply:
The test is expecting `'null'` to pass. This did initially strike me as a little odd. I haven't come across an instance when it hasn't defaulted to `None` correctly.

Reply:
Ahh just figured it out `None` = `'null'` when `json.dump`ed.